ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -183,6 +185,7 @@ private void setHeadResponseHeaders(BlobInfo blobInfo, RestResponseChannel restR
     if (blobProperties.getContentType() != null) {
       restResponseChannel.setHeader(RestUtils.Headers.CONTENT_TYPE, blobProperties.getContentType());
     }
+    restResponseChannel.setHeader(RestUtils.Headers.ACCEPT_RANGES, "bytes");
```

Comment:
Will use a constant for bytes. 
I think that HEAD should return accept-ranges since it is supposed to show what headers would be returned in a standard get request.